src/pages/ProfileInfoPage.jsx

```diff
@@ -0,0 +1,208 @@
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faFacebook,
+  faTwitter,
+  faLinkedin,
+  faYoutube,
+} from "@fortawesome/free-brands-svg-icons";
+import {
+  faLaptop,
+  faLanguage,
+  faBookOpen,
+  faArchive,
+} from "@fortawesome/free-solid-svg-icons";
+
+import ButtonProfile from "../shared/ButtonProfile";
+import profileImg from "../assets/images/profile/profile.png";
+
+const DEFAULT_DATA = {
+  firstName: "",
+  secondName: "",
+  description: "",
+  jop: "",
+  websiteUrl: "",
+  facebookUrl: "",
+  twitterUrl: "",
+  linkedinUrl: "",
+  youtubeUrl: "",
+};
+
+const socialNetwork = [
+  {
+    name: "Website",
+    icon: faLaptop,
+    key: "websiteUrl",
+    bg_color: "EC0707",
+    color: "gray",
+  },
+  {
+    name: "Facebook Profile",
+    icon: faFacebook,
+    key: "facebookUrl",
+    bg_color: "EC0707",
+    color: "0A48E0",
+  },
+  {
+    name: "Twitter Profile",
+    icon: faTwitter,
+    key: "twitterUrl",
+    bg_color: "EC0707",
+    color: "10A7D7",
+  },
+  {
+    name: "LinkedIn Profile",
+    icon: faLinkedin,
+    key: "linkedinUrl",
+    bg_color: "EC0707",
+    color: "0A48E0",
+  },
+  {
+    name: "YouTube Profile",
+    icon: faYoutube,
+    key: "youtubeUrl",
+    bg_color: "red",
+    color: "white",
+  },
+];
+
+const ProfileInfoPage = () => {
+  const inputStyle = "p-2 border-2 border-current md:p-4 focus:outline-none";
+  const iconStyle = "w-3 h-3 mx-2 text-lg text-gray-500 md:h-5 md:w-5";
+
+  const [formState, setFormState] = useState(DEFAULT_DATA);
+
+  const handleChange = (event, key) => {
+    setFormState({ ...formState, [key]: event.target.value });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(formState);
+  };
+
+  return (
+    <section className="relative pb-8 border border-b-0 border-gray-400">
+      <div className="flex flex-wrap justify-between gap-2 p-4">
+        <div className="flex">
+          <img src={profileImg} alt="profile image" className="w-20 mr-3" />
+          <div>
+            <h1 className="font-bold">Ahmed Ashraf</h1>
+            <p className="text-gray-500">No risk No Fun</p>
+            <p className="text-gray-500">Jop tittle</p>
+          </div>
+        </div>
+        <div>
+          <p>
+            <FontAwesomeIcon icon={faBookOpen} className={iconStyle} />
+            <span className="font-bold text-blue-600"> 4 Learning</span>
```

Comment:
All of these things must be dynamic, store them in the user data array and render them dynamic.